### ISSUE TITLE
Make it responsive

### ISSUE THREAD

**Issue by Raghwendra-Dey** [Raghwendra-Dey/productivity_meter#1]
This project is not made with keeping responsiveness in mind. And thus the view of this project in mobile phones is not that good. Make it responsive to be able to use in mobile phones as well with same ease. Also if you have a good sense of designing then do consider to give it some more styling to improve the look.

**Comment by LazyCoder-1506** [comment]
I would like to contribute to the styling and responsiveness

**Comment by Raghwendra-Dey** [comment]
@LazyCoder-1506 please go ahead. Presently this doesn't look good in the mobile devices. Please change the CSS to make it compatible with phones. Along with that, if you want to give it some more styling, then please do so.

**Comment by LazyCoder-1506** [comment]
Yes sure. I am pretty comfortable with Bootstrap, so if you allow I want to use Bootstrap for styling and making it responsive

**Comment by KeertimaanNayak** [comment]
I am willing to make the page responsive and look into the styling of this page.

**Comment by ananya8226** [comment]
I would like like to make it more responsive and improve the styling of this page, making it more user friendly.

**Comment by Raghwendra-Dey** [comment]
@ananya8226 and @KeertimaanNayak you both are welcomed to contribute towards improving the styling of the page, and making it responsive and user friendly. Since, this project needs a lot of styling works, `a team work is always better than one person working alone`. So, I would suggest you three to collaborate on making this project responsive and give it more styling. The designing aspects decisions you guys make among yourselves, and I am always there to oversee it. I suggest firstly you should focus on discussing and finalising a design and make it in some drawing software, and then after agreeing to a final design we can distribute the implementation works. @LazyCoder-1506 was suggesting to use bootstrap. If he has started working on it, You guys can pitch him about the status and collaborate accordingly. Please let me know if you three need any help from my side. Also after finalising the design please any one of you let me know about it.

**Comment by Nishant2907** [comment]
Hello, i am into adding responsiveness to a website by using bootstrap. Can i also work with pure css :)

**Comment by Raghwendra-Dey** [comment]
@Nishant2907 please go ahead. I have added you all to a chat room, you guys can start discussing about the design ideas there.

**Comment by Raghwendra-Dey** [comment]
@Nishant2907 Please join the [gitter channel](https://gitter.im/productivity_meter/community) and introduce yourself to the community there. Also, I have sent you invite for the `UI_works` chat room, please join that.

**Comment by ghost** [comment]
@Raghwendra-Dey  Sir, I want to work on this issue to make this website responsive for different devices.

Thanks & Regards
Urvashi Anand

**Comment by Raghwendra-Dey** [comment]
@urvashi-code1255 you are welcomed to contribute towards responsiveness of this project. Currently we are thinking of using bootstrap for this purpose. Please join the [gitter channel](https://gitter.im/productivity_meter/community) and introduce yourself to the community. Also please join [this chat room](https://gitter.im/productivity_meter/UI_works) to participate in the discussions of the UI aspects of the project.

**Comment by kri-bhawna** [comment]
@Raghwendra-Dey  Sir, I also want to contribute towards styling and responsiveness using bootstrap.
Bhawna

**Comment by inumausam** [comment]
Sir, I would also like to contribute towards this

**Comment by Raghwendra-Dey** [comment]
@kri-bhawna @inumausam you are welcomed. Please join the relevant channels. And read through the discussions of the UI_works chatroom and feel free to give your inputs.

**Comment by Vishal-raj-1** [comment]
@Raghwendra-Dey I would like to work on this issue!! I will use bootstrap for that purpose kindly assign it to me !! I have already join the UI_Works Glitter Room.

**Comment by Raghwendra-Dey** [comment]
> @Raghwendra-Dey I would like to work on this issue!! I will use bootstrap for that purpose kindly assign it to me !! I have already join the UI_Works Glitter Room.

Actually, currently we are working on this in part by part basis. We be opening more issues soon, once the currently assigned tasks are done.

**Comment by Vishal-raj-1** [comment]
> Actually, currently we are working on this in part by part basis. We be opening more issues soon, once the currently assigned tasks are done.

Okay!! No Problem
